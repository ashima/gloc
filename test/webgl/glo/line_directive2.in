#line 2
/*if*/#if defined E
/*A */#/*very commented */line/*line directive */10/* */77// with trailing comment
int foo() {
#line 13 3
  return 1;
}
#/*elif*/elif defined F // comments trailing pp conditional
#line 13 4
int foo() {
  return 2;
}
#else
int foo() {
  return 3;
}
#endif // comments trailing endif
