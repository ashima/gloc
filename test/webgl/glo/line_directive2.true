{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": { "copyright": [ 2012, [ "", "" ] ] },
  "units": [
    {
      "outsym": [ "foo" ],
      "source":
        "#line 1 GLOC_0\n#line 2\n/*if*/#if defined E\n/*A      very commented */ #line                 10/**/ GLOC_77/*  with trailing comment*/\nint foo() {\n#line 13 GLOC_3\n  return 1;\n}\n#else\n#line 15\n/* elif*/ #if defined F/*  comments trailing pp conditional*/\n#line 13 GLOC_4\nint foo() {\n  return 2;\n}\n#else\nint foo() {\n  return 3;\n}\n#endif/*  comments trailing endif*/\n#line 20\n#endif/*  comments trailing endif*/"
    }
  ],
  "linkmap": {
    "3": "<stdin>#n=3",
    "77": "<stdin>#n=77",
    "4": "<stdin>#n=4",
    "0": "<stdin>#n=0"
  }
}
