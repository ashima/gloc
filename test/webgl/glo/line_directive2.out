{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "copyright": [ "", 2011, "" ],
    "author": {},
    "license": null,
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [],
      "outsym": [ "foo" ],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source":
        "#line 1 GLOC_0\n#line 2\n/*if*/#if defined E\n/*A      very commented */ #line                 10/**/ GLOC_77/*  with trailing comment*/\nint foo() {\n#line 13 GLOC_3\n  return 1;\n}\n#else\n#line 15\n/* elif*/ #if defined F/*  comments trailing pp conditional*/\n#line 13 GLOC_4\nint foo() {\n  return 2;\n}\n#else\nint foo() {\n  return 3;\n}\n#endif/*  comments trailing endif*/\n#line 20\n#endif/*  comments trailing endif*/"
    }
  ],
  "linkmap": {
    "3": "<stdin>#3",
    "77": "<stdin>#77",
    "4": "<stdin>#4",
    "0": "<stdin>#0"
  }
}
